solace/operation.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://asyncapi.com/bindings/solace/operation.json",
+  "title": "Operation Schema",
+  "description": "This object contains information about the operation representation in Solace.",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "bindings": {
+      "description": "The inner bindings",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "destinationType": {
+            "type": "string",
+            "enum": [
+              "queue",
+              "topic"
+            ],
+            "description": "If the type is queue, then the subscriber can bind to the queue, which in turn will subscribe to the topic as represented by the channel name."
+          },
+          "deliveryMode": {
+            "type": "string",
+            "enum": [
+              "direct",
+              "persistent"
+            ]
+          },
+          "queue": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "description": "The name of the queue"
+              },
+              "topicSubscriptions": {
+                "type": "array",
+                "description": "The list of topics that the queue subscribes to.",
+                "items": {
+                  "type": "string"
+                }
+              },
+              "exclusive": {
+                "type": "boolean"
+              }
+            }
+          }
+        }
```

Comment:
If `queue` is only valid with `destinationType: queue` then we can use the `oneOf` to have more restricted validation:

```json
      "items": {
        "type": "object",
        "properties": {
          "deliveryMode": {
            "type": "string",
            "enum": [
              "direct",
              "persistent"
            ]
          }
        },
        "oneOf": [
          {
            "properties": {
              "destinationType": {
                "type": "string",
                "const": "queue",
                "description": "If the type is queue, then the subscriber can bind to the queue, which in turn will subscribe to the topic as represented by the channel name."
              },
              "queue": {
                "type": "object",
                "properties": {
                  "name": {
                    "type": "string",
                    "description": "The name of the queue"
                  },
                  "topicSubscriptions": {
                    "type": "array",
                    "description": "The list of topics that the queue subscribes to.",
                    "items": {
                      "type": "string"
                    }
                  },
                  "exclusive": {
                    "type": "boolean"
                  }
                }
              }
            }
          },
          {
            "properties": {
              "destinationType": {
                "type": "string",
                "const": "topic",
                "description": "If the type is queue, then the subscriber can bind to the queue, which in turn will subscribe to the topic as represented by the channel name."
              }
            }
          }
        ]
      }
```

Reply:
That's a good idea, I'll consider it.